pkg/controller/migmigration/backup.go

```diff
@@ -63,6 +67,55 @@ func (t *Task) ensureBackup() error {
 	return nil
 }
 
+// Ensure the second backup on the source cluster has been created and has the
+// proper settings.
+func (t *Task) ensureCopyBackup() error {
```

Comment:
I agree with the basic point here -- "Copy" in `ensureCopyBackup` looks like a verb rather than an adjective here. If it's important to keep the word "Copy" in there, then `BackupForCopy` would be better. I think either of the suggested options "with/withoutPVs" or "1/2" work just fine. I'm not sure that I have an opinion on which is better.

Reply:
Certainly not opposed to renaming it. If others agree on this I will happily change it.